modules/cactus-web/src/List/List.tsx

```diff
@@ -0,0 +1,112 @@
+import * as icons from '@repay/cactus-icons'
+import PropTypes from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+import { margin, MarginProps } from 'styled-system'
+
+import Flex from '../Flex/Flex'
+import Text from '../Text/Text'
+
+interface ListProps
+  extends MarginProps,
+    React.DetailedHTMLProps<React.HTMLAttributes<HTMLUListElement>, HTMLUListElement> {
+  divided?: boolean
+}
+
+interface ListItemProps
+  extends React.DetailedHTMLProps<React.HTMLAttributes<HTMLLIElement>, HTMLLIElement> {
+  icon?: keyof typeof icons
+  header?: React.ReactNode
+  headerAs?: keyof JSX.IntrinsicElements | React.ComponentType<any>
+}
+
+const UL = styled.ul<{ $divided: boolean }>`
+  padding: 0;
+  margin: 0;
+  list-style-type: none;
+  ${margin}
+
+  & > li > .item-flex > .item-inner-flex > ul {
```

Comment:
This worked just fine when I did it like this, was there another reason for doing it the long way? (I guess you could also replace the second `&` with `ul`, but after the recent Modal close button issue I figured more specific is better.)
```suggestion
  & & {
```

Reply:
It's been a while since I wrote the original css for this (back when i was working on the channels ticket) so I can't remember if I had issue doing it this way or not, but I will test it out.